thrust/thrust/detail/allocator_aware_execution_policy.h

```diff
@@ -27,7 +27,7 @@ _CCCL_IMPLICIT_SYSTEM_HEADER
 #include <thrust/detail/alignment.h>
 
 #include <type_traits>
-
+#include <type_traits>
```

Comment:
Urgh sorry, that seems to have been an issue with the UI, I accidentally added the header
```suggestion
```